spec/javascripts/components/layout-super-navigation-header-spec.js

```diff
@@ -328,23 +328,29 @@ describe('The super header navigation', function () {
     var $navLinks
     var $firstNavLink
     var $lastNavLink
+    var $searchMenu
+    var $searchTabable
+    var $lastSearchTabable
 
     beforeEach(function () {
       thisModule.init()
       $navMenuButton = document.querySelector('#super-navigation-menu-toggle')
       $searchMenuButton = document.querySelector('#super-search-menu-toggle')
       $navMenu = document.querySelector('#super-navigation-menu')
-      $navLinks = $navMenu.querySelectorAll('li')
-      $firstNavLink = $navLinks[0].querySelector('a')
-      $lastNavLink = $navLinks[$navLinks.length - 1].querySelector('a')
+      $navLinks = $navMenu.querySelectorAll('li a')
+      $firstNavLink = $navLinks[0]
+      $lastNavLink = $navLinks[$navLinks.length - 1]
+      $searchMenu = document.querySelector('#super-search-menu')
+      $searchTabable = $searchMenu.querySelectorAll('li a, input, button')
```

Comment:
Small nitpick, can we change the variable name here to be the same as the main JS file, so it would change from `searchTabbable` to `searchMenuTabbable`